tests/smoke/test_launch_states.py

```diff
@@ -25,9 +25,9 @@
 @pytest.fixture
 def args():
     # Test with PRPC node
-    args = Args(initial_cycle=10, reward_data_provider='tzkt', api_base_url='https://api.carthage.tzkt.io/v1/')
+    args = Args(initial_cycle=10, reward_data_provider='tzkt', api_base_url='https://api.hangzhounet.tzkt.io/v1/')
```

Comment:
This needs to be the URL from constants.py ? it was hangzhou2net and I changed it